[FW-141] Update README (#477)
HyperloopUPV-H8/ST-LIB: README.md

It is recommended to set up an `STLIB_PATH` env variable pointint to the root directory where the ST-LIB is located so that other team tools work out of the box. Many tutorials exist on how to set an environment variable in different OSs, you should easily find one by googling it.

## Container Setup
To use it you must install [Dev Containers extension on VSCode](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) and [Docker](https://www.docker.com). Be careful to use the instructions related to your OS, as docker not works in the same way exactly between them.

> Then, when you open this folder in VSCode, you will have the ability to reopen it inside the container. Don't worry, the first time you do it takes a loong time.

Reviewer: ```suggestion
Then, when you open this folder in VSCode, you will have the ability to reopen it inside the container. Don't worry, the first time you do it will take some time.
```
